tests/test_vallox_sensors.py

```diff
@@ -0,0 +1,167 @@
+"""Tests for Vallox WebSocket API sensor control functionality."""
+from unittest import mock
+
+import pytest
+
+from vallox_websocket_api import (
+    Profile,
+    CellState,
+    SupplyHeatingAdjustMode,
+    DefrostMode,
+    ValloxInvalidInputException,
+)
+
+
+@pytest.mark.asyncio
+async def test_set_rh_sensor_control(vallox, ws):
+    """Test setting RH sensor control."""
+    vallox.set_values = mock.AsyncMock()
+
+    # Test HOME profile
+    await vallox.set_rh_sensor_control(Profile.HOME, True)
+    vallox.set_values.assert_called_once_with({"A_CYC_HOME_RH_CTRL_ENABLED": True})
+    vallox.set_values.reset_mock()
+
+    # Test AWAY profile
+    await vallox.set_rh_sensor_control(Profile.AWAY, False)
+    vallox.set_values.assert_called_once_with({"A_CYC_AWAY_RH_CTRL_ENABLED": False})
+
+    # Test invalid profile
+    with pytest.raises(ValloxInvalidInputException):
+        await vallox.set_rh_sensor_control(Profile.FIREPLACE, True)
```

Comment:
Alternatively you can make this test parametric to try all settings at least once. Then invalid profile one will need to move to a separate method.

```suggestion
@pytest.mark.asyncio
@pytest.mark.parametrize("profile, enabled, expected_call", [
    (Profile.HOME, True, {"A_CYC_HOME_RH_CTRL_ENABLED": True}),
    (Profile.AWAY, False, {"A_CYC_AWAY_RH_CTRL_ENABLED": False}),
    (Profile.BOOST, False, {"A_CYC_BOOST_RH_CTRL_ENABLED": False}),
])
async def test_set_rh_sensor_control(vallox, profile, enabled, expected_call):
    """Test setting RH sensor control."""
    vallox.set_values = mock.AsyncMock()

    await vallox.set_rh_sensor_control(profile, enabled)
    vallox.set_values.assert_called_once_with(expected_call)


@pytest.mark.asyncio
async def test_set_rh_sensor_control_invalid_profile(vallox):
    """Test setting RH sensor control with invalid profile."""
    vallox.set_values = mock.AsyncMock()

    with pytest.raises(ValloxInvalidInputException):
        await vallox.set_rh_sensor_control(Profile.FIREPLACE, True)
```

Reply:
Fixed. Parameterized tests to cover all possible values and increase coverage.